skiller.sh

```diff
@@ -135,6 +135,48 @@ def create_skill():
     return $class_name()
 """ > "$skill_dir/__init__.py"
 
+echo """{
+	\"name\": \"${capital_desc}\",
```

Comment:
For readability, you should be able to use a single quote on the outside (the part that is `"""` right now) so that you don't have to backslash the quotes. To add a variable replacement, surround it with another single end-quote and start-quote. Here's an example:
```bash
echo '{
    "name": "'${capital_desc}'",
    "skillMetadata": {
        ...
}'
```

Reply:
Ah yes, that looks much better. I was trying to match style of the rest of the script, but this would be much cleaner.

Reply:
As a side note, I just realized I don't think those triple quotes actually do anything (it's really just `""` next to `"something`).